src/shogun/clustering/Hierarchical.cpp

```diff
@@ -212,8 +216,86 @@ SGMatrix<int32_t> CHierarchical::get_cluster_pairs()
 	return SGMatrix<int32_t>(pairs,2,merges, false);
 }
 
-
 void CHierarchical::store_model_features()
 {
-	/* TODO. Currently does nothing since apply methods are not implemented. */
+	CDenseFeatures<float64_t>* rhs =
+	    dynamic_cast<CDenseFeatures<float64_t>*>(distance->get_rhs());
+
+	int32_t num_vectors = rhs->get_num_vectors();
+	int32_t num_features = rhs->get_num_features();
+
+	int32_t* n_cluster_samples = SG_MALLOC(int32_t, num_vectors + merges);
+	SGVector<int32_t>::fill_vector(n_cluster_samples, num_vectors + merges, 0);
```

Comment:
use `linalg::set_const` as per https://github.com/shogun-toolbox/shogun/issues/2747

Reply:
Looking into it

Reply:
Made the change :)